src/GitHub.App/Extensions/AkavacheExtensions.cs

```diff
@@ -34,7 +38,7 @@ public static class AkavacheExtensions
         {
             return Observable.Defer(() =>
             {
-                var absoluteExpiration = blobCache.Scheduler.Now + maxCacheDuration;
+                var absoluteExpiration = blobCache.Scheduler.Now.ToUniversalTime() + maxCacheDuration;
```

Comment:
Given that `Scheduler.Now` returns a `DateTimeOffset`, the call to `ToUniversalTime` is unnecessary. The value returned represents the _same point in time_ as the original value. Here's some code to demonstrate.

```csharp
var localTime = new DateTimeOffset(new DateTime(2007, 6, 15, 
12, 0, 0), TimeSpan.FromHours(8));
var universalTime = localTime.ToUniversalTime();
Console.WriteLine("Local offset: " + localTime.Offset);
Console.WriteLine("Universal offset: " + universalTime.Offset
);
Console.WriteLine("Difference: " + (localTime - universalTime
));
```

__Output__
```
Local offset: 08:00:00
Universal offset: 00:00:00
Difference: 00:00:00
```